test/e2e_flags.go

```diff
@@ -33,6 +33,8 @@ const (
 	// namespace tests in.
 	AlternativeServingNamespace = "serving-tests-alt"
 
+	// side car injected ns
+	SideCarServingNamespace = "serving-tests-sidecar-enabled"
```

Comment:
`serving-tests` and `serving-tests-alt` namespaces also enabled side car injection on Mesh mode.

https://github.com/knative/serving/blob/28f2a2460ee8fe89cca025d0831380ea7fe0df04/test/e2e-common.sh#L433-L435

So, it would be better to use either of the namespace. I wonder that we can merge your config into https://github.com/knative/serving/blob/master/test/config/mtls/policy.yaml. Then, we should change the directory name to `security` or something from `/auth`, `/mtls`.

Reply:
Absolutely, was not sure of it. I will make the change accordingly. If I change the directory name to security, what are the scripts i need to change

Reply:
i have moved the files from mtls to security folder and made the changes in e2e-common.sh 
@nak3 Please review. And the tests are failing on istio client dependency which i had referred in the clients.go which i later reverted based on your suggestion. Not sure why the tests are still referring to it.